platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionError.java

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.mapboxsdk.offline;
+
+/**
+ * Created by antonio on 2/21/16.
+ */
+public class OfflineRegionError {
+
+    /**
+     * Success = 1
```

Comment:
That still depends on the definition of `Response::Error::Reason`. Using numeric values in the Java interface is fine. They key point is that we need to use _identifiers_ like `Response::Error::Reason::NotFound` to translate to the numeric values that the Java API uses, not just rely on the _values_ of those identifiers lining up. [Same idea as what you do here.](https://github.com/mapbox/mapbox-gl-native/blob/0cc6a9b68668a9d10d645fc389233435b074a382/platform/android/src/jni.cpp#L2036-L2044)